src/routes/news/fetchMoreData.js

```diff
@@ -0,0 +1,40 @@
+// @flow
```

Comment:
I also think this should be moved to `app`. Has to be discuessed with @maxammann though.